src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,40 @@
      * Андрей - 600
      * София - 900</p>
      */
+
+    static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        if (LocalDate.parse(dateFrom, DATE_FORMAT)
+                .compareTo(LocalDate.parse(dateTo, DATE_FORMAT)) > 0) {
+            throw new IllegalDateParametersException("Wrong parameters");
+        }
+
+        int sum = 0;
```

Comment:
create this variable inside of for loop and you won't need to refresh it manually